internal/queryrun/repository.go

```diff
@@ -61,6 +63,15 @@ func (r *Repo) Get(id string) (*Run, error) {
 	}
 
 	run := new(Run)
+
+	// Done because UnmarshalMap can't unmarshal in interface{}
+	var database string
+	attributevalue.Unmarshal(out.Item["Database"], &database)
+	// TODO: add switch-case in the future
+	if database == "clickhouse" {
```

Comment:
Let's use a predefined database type here (stored as enum). You can move runsettings.type to this package.

Reply:
`runsetting` package already imports `queryrun` package, so if I move type, there will be cycle in imports. I guess we can leave runsettings.type here.

Reply:
Ok, I created directory database and move runsettings in it